walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,99 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
+import java.util.Stack;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // Cases
+    // Options for no attendees - should be available all day.
+    if (request.getAttendees().size() == 0) {
```

Comment:
Using .isEmpty() to check may be better. (The same suggestions are given in some other places, which is also specified by Robert.)

Reply:
Done